static.html

```diff
@@ -12,7 +12,9 @@
 <body>
   <div id="root">
     <!-- insert your tags here -->
+  
   </div>
+  <script src="robot.js"></script>
```

Comment:
The assignment asked for you to hand-code the HTML that the robot.js file creates dynamically. Don't use a script tag here. I'll point you to a message in slack where this is discussed in more detail.

Reply:
oh, yes. i understood just now. now I correct the code and commit and pull request again.